src/infrastructure/tauri/ProjectRepository.ts

```diff
@@ -104,15 +102,15 @@ export const mkProjectRepositoryTauri = (): task.Task<ProjectRepository> => {
               taskOption.chainOptionK(projectPrism.local.getOption),
             ),
           }),
-          taskOption.chainTaskEitherK(({ rootDir, project }) =>
+          taskOption.chainTaskK(({ rootDir, project }) =>
             path.join(rootDir, project.value.basePath),
           ),
         );
 
         const removeProjectDir: taskEither.TaskEither<Array<string>, void> = pipe(
           getProjectDir,
-          taskOption.chainTaskEitherK(api.removeDir),
-          taskEither.fromTaskOption(() => ['Could not remove project dir']),
+          taskEither.fromTaskOption(() => ['Could not get project dir']),
+          taskEither.chain(flow(api.removeDir, taskEither.mapLeft(array.of))),
```

Comment:
Gibt es eine Regel, welche Fehler wir auswerten?

In `getProjectDir` (`path.join`) spülen wir die Fehlermeldungen. Hier emittieren wir den Fehler dann doch wieder, aber mit einer generischen Meldung ('Could not get project dir'). Fehler in `api.removeDir` leiten wir weiter … Es erscheint etwas inkonsistent, aber vielleicht verstehe ich das Prinzip noch nicht :)